website/members/apps.py

```diff
@@ -64,7 +64,12 @@ def announcements(self, request) -> list[dict]:
         announcements = []
         if request.member and not request.member.has_active_membership():
             announcements.append(
-                {"rich_text": render_to_string("members/announcement_not_member.html")}
+                {
+                    "rich_text": render_to_string(
+                        "members/announcement_not_member.html",
+                        context={"member": request.member},
```

Comment:
I think (but not 100% sure) that this is already available by default through a context processor

Reply:
I mean I don't see the harm in explicitly giving it in the context? Imo is a bit more readable anyway